schemas/base-metadata.schema.ts

```diff
@@ -5,6 +5,14 @@ export const preprocessMetadataSchema = (schema: any = baseMetadataSchema) => {
     // Add unit to weight
     schema.properties.Subject.properties.weight.unit = 'kg'
 
+    schema.properties.Subject.properties.sex.enumLabels = {
+        O: 'Other',
+        M: 'Male',
+        F: 'Female',
+        U: 'Unknown'
```

Comment:
```suggestion
        M: 'Male',
        F: 'Female',
        U: 'Unknown',
        O: 'Other'
```

The order here appears to adjust the order in the dropdown; ordering by frequency

Reply:
Good catch. Thanks!